SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -252,6 +252,10 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (ConfigRestartRequired = false, DisplayName = "Stop on exit"))
 	bool bStopSpatialOnExit;
 
+	/** Start a local SpatialOS deployment when clicking 'Play'. */
+	UPROPERTY(EditAnywhere, config, Category = "Launch", meta = (ConfigRestartRequired = false, DisplayName = "Auto-start local deployment"))
```

Comment:
Is it intentional that the names are different: "Auto-start local deployment" and bAutoStartSpatialDeployment?

Reply:
Renamed to bAutoStartLocalDeployment